erpnext/payroll/doctype/payroll_entry/payroll_entry.py

```diff
@@ -489,6 +517,25 @@ def create_salary_slips_for_employees(employees, args, publish_progress=True):
 			if publish_progress:
 				frappe.publish_progress(count*100/len(set(employees) - set(salary_slips_exists_for)),
 					title = _("Creating Salary Slips..."))
+		else:
+			salary_slip_name = frappe.db.sql(
+				'''SELECT 
+						name 
+					FROM
+						`tabSalary Slip` 
+					WHERE 
+						company=%s
+					AND 
+						start_date >= %s 
+					AND 
+						end_date <= %s
+					AND 
+						employee = %s
```

Comment:
We generally follow below formatting:
```suggestion
						company=%s
						and start_date >= %s 
						and end_date <= %s
						and employee = %s
```